sgl-kernel/benchmark/bench_fp8_blockwise_gemm.py

```diff
@@ -0,0 +1,175 @@
+import argparse
+import copy
+import itertools
+
+import torch
+import triton
+from sgl_kernel import fp8_blockwise_scaled_mm
+from vllm._custom_ops import cutlass_scaled_mm as vllm_scaled_mm
+
+
+def to_int8(tensor: torch.Tensor) -> torch.Tensor:
+    return torch.round(tensor.clamp(min=-128, max=127)).to(dtype=torch.int8)
+
+
+WEIGHT_SHAPES = {
```

Comment:
I think DeepSeek V3 is sufficient, other models use per-tensor FP8, so we don't need a benchmark in that form.